app/constants/errorCode.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Map error codes to messages. HTML is allowed in messages.
+ */
+const map = {
+  e001: 'Data formatting error; see below for details.',
+};
+
+/**
+ * Return error message for use with dangerouslySetInnerHTML attribute
+ */
+export default function(code, defaultMsg = 'Unknown error.') {
```

Comment:
probably overthinking it but seems like `defaultMsg` is the only thing here that potentially could allow non-static / theoretically possibly dangerous content -- is this param necessary?